include/deal.II/lac/precondition.h

```diff
@@ -1842,7 +1842,8 @@ PreconditionChebyshev<MatrixType,VectorType>::initialize
               ExcMessage ("Need to set at least two iterations to find eigenvalues."));
 
       // set a very strict tolerance to force at least two iterations
-      ReductionControl control (data.eig_cg_n_iterations, 1e-35, 1e-10);
```

Comment:
Well, not really. I just noticed that the test results contain output from this solve operation which is not expected to converge.